packages/compass-indexes/src/components/create-index-actions/create-index-actions.tsx

```diff
@@ -90,25 +79,25 @@ function CreateIndexActions({
         {renderInProgress()}
       </div>
       <div className={modalFooterActionsStyles}>
-        <Button
-          data-testid="create-index-actions-create-index-button"
-          onClick={onConfirm}
-          variant="primary"
-          className={createIndexButtonStyles}
-          darkMode={darkMode}
-        >
-          Create Index
-        </Button>
+        {!inProgress && (
+          <Button
+            data-testid="create-index-actions-create-index-button"
+            onClick={onConfirm}
+            variant="primary"
+            className={createIndexButtonStyles}
+          >
+            Create Index
+          </Button>
+        )}
         <Button
           data-testid="create-index-actions-cancel-button"
-          darkMode={darkMode}
           onClick={onCancel}
         >
-          Cancel
+          {inProgress ? 'Close' : 'Cancel'}
```

Comment:
We added Cancel to this button, but we do not actually cancel the op (of index creation).

Reply:
I think we use the "Cancel" everywhere to cancel the modal (eg create db or collection modals), but here I agree with Maurizio, that cancel sounds confusing because we can not cancel the creating index that is in progress, therefore in his case, we replace it with the "Close" button.